<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuWindow</class>
 <widget class="QWidget" name="MainMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainMenuWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QPushButton" name="newgameButton">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 700 16pt &quot;Verdana&quot;;
	border:none;
    color: white;
}

QPushButton:hover {
    color: rgb(75, 88, 67);
}</string>
       </property>
       <property name="text">
        <string>New Game</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QPushButton" name="findserversButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border:none;
	font: 700 16pt &quot;Verdana&quot;;
            color: white;
}

QPushButton:hover {
    color: rgb(75, 88, 67);
}</string>
       </property>
       <property name="text">
        <string>Find Servers</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QPushButton" name="quitButton">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 700 16pt &quot;Verdana&quot;;
	border:none;
            color: white;
}

QPushButton:hover {
    color: rgb(75, 88, 67);
}</string>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
